portal/translations/es_ES/LC_MESSAGES/messages.po

```diff
@@ -2591,665 +2613,669 @@ msgstr "No se encontró ninguna identidad"
 msgid "User Not Found"
 msgstr "Usuario no encontrado"
 
-#: https://github.com/uwcirg/truenth-portal/tree/develop/portal/views/portal.py#L1194
+#: https://github.com/uwcirg/truenth-portal/tree/develop/portal/views/portal.py#L1193
 #, python-format
 msgid "I consent to sharing information with %(org_name)s"
 msgstr "Doy mi consentimiento para compartir información con %(org_name)s"
 
-#: https://github.com/uwcirg/truenth-portal/tree/develop/portal/views/reporting.py#L114
+#: https://github.com/uwcirg/truenth-portal/tree/develop/portal/views/reporting.py#L115
 msgid "TOTAL"
 msgstr "TOTAL"
 
-# Intervention: assessment_engine
-msgid "Assessment Engine"
-msgstr ""
-
-# Intervention: decision_support_wisercare
-msgid "Decision Support WiserCare"
-msgstr "Apoyo de decisión WiserCare"
-
-# AppText: IRONMAN staff registraion email URL
-msgid "{config[LR_ORIGIN]}/c/portal/truenth/asset/mail?version=latest&uuid=e4206748-f26c-b8b2-f6b9-b67465b6f746&editorUrl=true"
-msgstr "{config[LR_ORIGIN]}/c/portal/truenth/asset/mail?version=latest&uuid=e4206748-f26c-b8b2-f6b9-b67465b6f746&editorUrl=true"
-
-# classification_enum: Baseline
-msgid "Baseline"
-msgstr "Referencia"
+# AppText: landing sub-title
+msgid " "
+msgstr " "
 
-# Organization: Macquarie University Hospital
-msgid "Macquarie University Hospital"
+# Organization: Department Of Urology, Princess Alexandra Hospital
+msgid "Department Of Urology, Princess Alexandra Hospital"
 msgstr ""
 
-# AppText: patient reminder email IRONMAN
-msgid "{config[LR_ORIGIN]}/c/portal/truenth/asset/mail?version=latest&uuid=e6e21c27-6bc1-c0c5-de58-bcce0ba63f34"
-msgstr ""
+# Role: content_manager
+msgid "Content Manager"
+msgstr "Gestor de contenidos"
 
-# Organization: Winship Cancer Institute Emory University
-msgid "Winship Cancer Institute Emory University"
+# Organization: Australian Prostate Cancer Research Centre
+msgid "Australian Prostate Cancer Research Centre"
 msgstr ""
 
-# Intervention: care_plan
-msgid "<p>Organization and support for the many details of life as a prostate cancer survivor</p>"
-msgstr "<p>Organización y apoyo para los múltiples detalles de la vida como sobreviviente del cáncer de próstata</p>"
-
```

Comment:
still a bunch of deletions, are they ok?

Reply:
Looks like that only got moved to [here](https://github.com/uwcirg/truenth-portal/pull/3185/files/2c9a998ad5f66275a803e78225eeaba67a10a9b1#diff-68c37b10c885be7e2fa3dee8ff2d5759R3198)

If you hover over header for each file you can see the insertions/deletions; I skimmed them and roughly all of them have net insertions

![image](https://user-images.githubusercontent.com/5333433/57041780-ec899d00-6c17-11e9-8b2b-957a2bf90dad.png)